docs/src/components/function-card.astro

```diff
@@ -15,7 +15,7 @@ const { func } = Astro.props;
 const tags = getTags(func);
 ---
 
-<Card id={func.name} className="scroll-mt-20">
+<Card id={func.name} className="scroll-mt-20 overflow-hidden">
```

Comment:
This overflow-hidden breaks scroll-mt-20 (scroll offset to compensate for the header height)